AppCenter/AppCenter/Internals/Channel/MSChannelGroupDefault.m

```diff
@@ -152,8 +161,9 @@ - (void)channel:(id<MSChannelProtocol>)channel
 - (void)channel:(id<MSChannelProtocol>)channel
               didSetEnabled:(BOOL)isEnabled
     andDeleteDataOnDisabled:(BOOL)deletedData {
-  [self enumerateDelegatesForSelector:@selector
-        (channel:didSetEnabled:andDeleteDataOnDisabled:)
+  [self enumerateDelegatesForSelector:@selector(channel:
+                                                    didSetEnabled:
+                                          andDeleteDataOnDisabled:)
```

Comment:
Formatting looks weird but I think it's correct.

Reply:
I used the formatter setup with the default LLVM format :/.